src/tests/EdgeID/transition_id_tests.rs

```diff
@@ -0,0 +1,105 @@
+        vec![
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E43".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E37".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E42".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E39".to_string())),
+                Box::new(TransitionID::Simple("E31".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E32".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E34".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E33".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E40".to_string())),
+                Box::new(TransitionID::Simple("E30".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E38".to_string())),
+                Box::new(TransitionID::Simple("E36".to_string()))
+            ),
+            TransitionID::Conjunction(
+                Box::new(TransitionID::Simple("E41".to_string())),
+                Box::new(TransitionID::Simple("E35".to_string()))
+            )
+            ]; "Conjunction HalfAdm1 and HalfAdm2")]
+    fn transition_id_test(
+        path: &str,
+        machineExpression: QueryExpression,
+        transition_ids: Vec<TransitionID>,
+    ) {
+        let mock_model = Box::new(machineExpression);
+        let mut expected_ids: HashSet<&TransitionID> = HashSet::from_iter(transition_ids.iter());
+        let (_, system) =
+            reachability_test_helper_functions::create_system_recipe_and_machine(*mock_model, path);
+        for loc in system.get_all_locations() {
+            for ac in system.get_actions() {
+                for tran in system.next_transitions(&loc, &ac) {
+                    println!("{}", &tran.id);
```

Comment:
Please do not print anything in tests

Reply:
Den mappe burde ikke engang existere xD, har fjernet den da der er samme test et andet sted som ikke printer noget ud. Dette var en gammel version som var sluppet igennem code review.

Reply:
Den mappe burde ikke engang existere xD, har fjernet den da der er samme test et andet sted som ikke printer noget ud. Dette var en gammel version som var sluppet igennem code review.